libbpf-rs/src/print.rs

```diff
@@ -1,10 +1,10 @@
 use crate::*;
-use std::fmt;
+use lazy_static::lazy_static;
 use std::io::{self, Write};
 use std::os::raw::c_char;
-use std::sync::atomic::{AtomicPtr, Ordering};
+use std::sync::Mutex;
 
-#[derive(Debug, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Copy)]
```

Comment:
Curious, why is Clone needed?

Reply:
get_print copies the (level, callback) tuple for its return value, and Copy requires Clone.